src/tmr/tmr.c

```diff
@@ -112,6 +114,43 @@ void tmr_poll(struct list *tmrl)
 }
 
 
+/**
+ * Get the timer jiffies in microseconds
+ *
+ * @return Jiffies in [us]
+ */
+uint64_t tmr_jiffies_us(void)
+{
+	uint64_t jfs;
+
+#if defined(WIN32)
+	LARGE_INTEGER li;
+	static LARGE_INTEGER freq;
+
+	if (!freq.QuadPart)
+		QueryPerformanceFrequency(&freq);
+
+	QueryPerformanceCounter(&li);
+	li.QuadPart *= 1000000;
+	li.QuadPart /= freq.QuadPart;
+
+	jfs = li.QuadPart;
+#else
+	struct timespec now;
+
+	if (0 != clock_gettime(CLOCK_MONOTONIC_RAW, &now)) {
+		DEBUG_WARNING("jiffies: clock_gettime() failed (%m)\n", errno);
+		return 0;
+	}
+
+	jfs  = (long)now.tv_sec * (uint64_t)1000000;
+	jfs += now.tv_nsec/1000;
+#endif
+
+	return jfs;
+}
```

Comment:
this is a useful function, is it possible to make it a separate PR ?

here is a similar function:

https://github.com/baresip/baresip/blob/master/src/timer.c#L29

some people also wanted to have clock_gettime.

after this patch have been merged, we can delete baresip/src/timer.c

Reply:
Done: https://github.com/baresip/baresip/pull/1141

I will update this PR.